PiwikTracker/PiwikTracker.swift

```diff
@@ -38,10 +39,13 @@ final public class PiwikTracker: NSObject {
     ///   - siteId: The unique site id generated by the server when a new site was created.
     ///   - queue: The queue to use to store all analytics until it is dispatched to the server.
     ///   - dispatcher: The dispatcher to use to transmit all analytics to the server.
-    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher) {
+    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher, suiteName: String? = nil) {
```

Comment:
Yes, I think I will do that. I will add a short comment that the users properties are strongly bound to the server url and site id, so whenever they are lost the settings are lost as well.